### PR TITLE
Feature/ch7211: VRF jpv2 crud

### PR DESCRIPTION
**Pull request by connorwstein**
- Moving vrf related db operations into the vrf package, following the pattern in jpv2 of "service packages contain an orm.go and models.go for all state related to that service"
- The secp256k1 pubkey type is a useful standalone type that I wanted to use in the VRFSpec, so pulled it into the secp256k1 package (if it remained in vrf, then you'd have a import cycle vrf->job->vrf)
- Combining all the job add tests into a table test
- Removing the single function from cltest/vrf (not worth bringing all the baggage of cltest into vrf for a single function)
- Pulling toml specs into go code, same reasoning as https://github.com/smartcontractkit/chainlink/pull/4407#discussion_r631345033

### CODE REVIEW

File: core/cmd/jobs_commands.go [hunk 1]
```diff
@@ -106,6 +106,10 @@ func (p JobPresenter) FriendlyCreatedAt() string {
 		if p.KeeperSpec != nil {
 			return p.KeeperSpec.CreatedAt.Format(time.RFC3339)
 		}
+	case presenters.CronJobSpec:
```
Comment (connorwstein) [inline core/cmd/jobs_commands.go:109]: cc @spooktheducks this seemed to be missing

File: core/cmd/jobs_commands.go [hunk 2]
```diff
@@ -106,6 +106,10 @@ func (p JobPresenter) FriendlyCreatedAt() string {
...
+		if p.KeeperSpec != nil {
```
Comment (samsondav) [inline core/cmd/jobs_commands.go:110]: Should this not be p.CronJobSpec?

File: core/store/migrations/0027_vrf_v2.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
+package migrations
+
+import "gorm.io/gorm"
+
+const (
+	up27 = `
+        CREATE TABLE vrf_specs (
+            id BIGSERIAL PRIMARY KEY,
+            public_key text NOT NULL,
+			coordinator_address bytea NOT NULL,
```
Comment (samsondav) [inline core/store/migrations/0027_vrf_v2.go:10]: Weird indentation here.

Also, check constraint on address length?

File: core/cmd/jobs_commands.go (hunk 1 above)
Comment (samsondav) [reply core/cmd/jobs_commands.go:109]: test?

**Review by samsondav — DISMISSED** [review]

File: core/cmd/jobs_commands.go (hunk 1 above)
Comment (connorwstein) [reply core/cmd/jobs_commands.go:109]: added

**Pull request merged**